best-practices/html.md

```diff
@@ -120,31 +119,6 @@ Quickly and easily ensure proper rendering of your content by declaring an expli
 
 Strive to maintain HTML standards and semantics, but not at the expense of practicality. Use the least amount of markup with the fewest intricacies whenever possible.
 
-<a name="html-attribute-order"></a>
-
-## Attribute order
-
-HTML attributes should come in this particular order for easier reading of code.
-
-- `class`
-- `id`, `name`
-- `data-*`
-- `src`, `for`, `type`, `href`
-- `title`, `alt`
-- `aria-*`, `role`
-
-Classes make for great reusable components, so they come first. Ids are more specific and should be used sparingly (e.g., for in-page bookmarks), so they come second.
-
-```html
-<a class="..." id="..." data-modal="toggle" href="#">
-  Example link
-</a>
-
-<input class="form-control" type="text" />
-
-<img src="..." alt="..." />
-```
```

Comment:
I don't know anyone who pays attention to the order of attributes on HTML/JSX elements in PRs, I think we should remove this. 👍